app/controllers/profiles_controller.rb

```diff
@@ -18,12 +17,12 @@ def update
 
   def destroy
     current_user.destroy
-    redirect_to users_url, flash: { warning: "User was successfully destroyed." }
+    redirect_to users_url, flash: { warning: t('user.destroyed') }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.